boards/x86/intel_adl/doc/up_squared_pro_7000.rst

```diff
@@ -0,0 +1,67 @@
+:orphan:
+
+.. _up_squared_pro_7000_board:
+
+UP SQUARED PRO 7000 board
+#########################
+
+Overview
+********
+
+UP Squared Pro 7000 is the 3rd generation of palm-sized developer board of
+UP Boards series. UP Squared Pro 7000 is powered by Intel Alder Lake N
+(Intel N-series Platform).
+
+For more information about Intel N-series Platform please refer to
+:ref:`intel_adl_n`.
+
+This board configuration enables kernel support for the UP Squared Pro 7000 boards.
+
+Hardware
+********
+
+General information about the board can be found at the `UP_SQUARED_PRO_7000`_ website.
+
+Connections and IOs
+===================
+
+Refer to the `UP_SQUARED_PRO_7000`_ website for more information.
+
+Programming and Debugging
+*************************
+Use the following procedures for booting an image for an UP SQUARED PRO 7000 board.
```

Comment:
don't know if that matters on doc generation, but an empty line before might be missing

Reply:
Should be OK, https://builds.zephyrproject.io/zephyr/pr/65019/docs/boards/x86/intel_adl/doc/up_squared_pro_7000.html